examples/create-react-app/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "react-app",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.1.0",
  "private": true,
```

We need this to be `private` to ensure that it can't be published